scipy/stats/_covariance.py

```diff
@@ -0,0 +1,142 @@
+from functools import cached_property
+
+import numpy as np
+
+
+__all__ = ["Covariance", "CovViaPrecision", "CovViaPSD"]
```

Comment:
```suggestion
__all__ = ["Covariance", "CovViaPrecision"]
```

Didn't mean to make that public. Maybe I'll add a `_` to the name.